Symfony/CS/Fixer/Contrib/NoBlankLinesBeforeNamespaceFixer.php

```diff
@@ -26,7 +26,7 @@ public function fix(\SplFileInfo $file, $content)
     {
         $tokens = Tokens::fromCode($content);
 
-        foreach (array_keys($tokens->findGivenKind(T_NAMESPACE)) as $index) {
+        foreach ($tokens->findGivenKind(T_NAMESPACE) as $index => $token) {
```

Comment:
Could you ping me on gitter if I forget to do it please. I can't do it right now - busy.